app/src/main/res/values/strings.xml

```diff
@@ -163,21 +163,20 @@
     <string name="privacy_policy_popup_text">Privacy policy notice (required by some app stores):\n\nNO DATA IS COLLECTED AT ALL, which anyone can confirm since our app is libre software.</string>
     <string name="accept">Accept</string>
     <string name="importCatima">Import from Catima</string>
-    <string name="importCatimaMessage">Find a file likely named <i>Catima.csv</i> to import.\nOr create it from the Import/Export menu of another Catima app by pressing "Export" there first.</string>
+    <string name="importCatimaMessage">Find a file likely named <i>Catima.csv</i> to import.\nOr create it from the "Import/Export" menu of another Catima app by pressing "Export" there first.</string>
     <string name="importFidme">Import from FidMe</string>
-    <string name="importFidmeMessage">Find a file likely named <i>fidme-export-request-xxxxxx.zip</i> to import, and then select the barcode types manually afterwards.\nOr create it from your FidMe profile by choosing "Data Protection" and then pressing "Extract my data" first.</string>
+    <string name="importFidmeMessage">Find a file likely named <i>fidme-export-request-xxxxxx.zip</i> to import, and select the barcode types manually afterwards.\nOr create it from your FidMe profile by choosing "Data Protection" and then pressing "Extract my data" first.</string>
     <string name="importLoyaltyCardKeychain">Import from Loyalty Card Keychain</string>
-    <string name="importLoyaltyCardKeychainMessage">Find a file most likely named <i>LoyaltyCardKeychain.csv</i> to import.\nOr create it from the Import/Export menu in Loyalty Card Keychain by pressing "Export" first.</string>
+    <string name="importLoyaltyCardKeychainMessage">Find a file most likely named <i>LoyaltyCardKeychain.csv</i> to import.\nOr create it from the "Import/Export" menu in Loyalty Card Keychain by pressing "Export" there first.</string>
     <string name="importStocard">Import from Stocard</string>
-    <string name="importStocardMessage">Find your Stocard .zip file to import, and then select the barcode types manually afterwards.\nOr get it by emailing support@stocardapp.com and asking for an export of your data first.</string>
+    <string name="importStocardMessage">Find your Stocard.zip file to import, and select the barcode types manually afterwards.\nOr get it by e-mailing support@stocardapp.com asking for an export of your data.</string>
```

Comment:
This space before .zip was actually intended, because when I asked my data Stocard gave me a file named 50e33e49-cfa0-49ad-a297-c1d655f72b01-sync.zip. So no Stocard in the filename at all.

Reply:
```suggestion
    <string name="importStocardMessage">Find your Stocard ***-sync.zip file to import, and select the barcode types manually afterwards.\nOr get it by e-mailing support@stocardapp.com asking for an export of your data.</string>
```